Source/SwiftLintFramework/Rules/QuickDiscouragedCallRule.swift

```diff
@@ -0,0 +1,107 @@
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "quick_discouraged_call",
+        name: "Quick Discouraged Call",
+        description: "Discouraged call inside 'describe' and/or 'context' block.",
+        kind: .style,
+        nonTriggeringExamples: QuickDiscouragedCallRuleExamples.nonTriggeringExamples,
+        triggeringExamples: QuickDiscouragedCallRuleExamples.triggeringExamples
+    )
+
+    public func validate(file: File,
+                         kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard
+            fileContainsQuickSpec(file: file),
+            kind == .call,
+            let name = dictionary.name,
+            let kindName = QuickCallKind(rawValue: name),
+            QuickCallKind.restrictiveKinds().contains(kindName)
+            else { return [] }
+
+        return violationOffsets(in: dictionary.enclosedArguments)
+            .map {
+                StyleViolation(ruleDescription: type(of: self).description,
+                               severity: configuration.severity,
+                               location: Location(file: file, byteOffset: $0))
+            }
+    }
+
+    // MARK: - Private
+
+    private func fileContainsQuickSpec(file: File) -> Bool {
+        return !file.structure.dictionary.substructure.filter { $0.inheritedTypes.contains("QuickSpec") }.isEmpty
+    }
+
+    typealias ViolationOffset = Int
+
+    private func violationOffsets(in substructure: [[String: SourceKitRepresentable]]) -> [ViolationOffset] {
+        return substructure.flatMap { dictionary -> [ViolationOffset] in
+            return dictionary.substructure.flatMap(toViolationOffsets)
+        }
+    }
+
+    private func toViolationOffsets(dictionary: [String: SourceKitRepresentable]) -> [ViolationOffset] {
+        guard
+            let kind = dictionary.kind,
+            let offset = dictionary.offset
+            else { return [] }
+
+        if SwiftExpressionKind(rawValue: kind) == .call,
+            let name = dictionary.name, QuickCallKind(rawValue: name) == nil {
+            return [offset]
+        }
+
+        guard SwiftExpressionKind(rawValue: kind) != .call else { return [] }
+
+        return dictionary.substructure.flatMap(toViolationOffset)
+    }
+
+    private func toViolationOffset(dictionary: [String: SourceKitRepresentable]) -> ViolationOffset? {
+        guard
+            let name = dictionary.name,
+            let offset = dictionary.offset,
+            let kind = dictionary.kind,
+            SwiftExpressionKind(rawValue: kind) == .call,
+            QuickCallKind(rawValue: name) == nil
+            else { return nil }
+
+        return offset
+    }
+}
+
+// MARK: - Private
+
+private enum QuickCallKind: String {
```

Comment:
Changing it to static let.

By the way, this could be a nice enhancement thru SwiftLint's code. For instance

```swift
static SyntaxKind.commentKeywordStringAndTypeidentifierKinds()
static SyntaxKind.commentAndStringKinds()
static SyntaxKind.commentKinds()
static SyntaxKind.allKinds()
```

and others could all be static lets.